java/RSPEC-2278.java

```diff
@@ -0,0 +1,7 @@
+import javax.crypto;
```

Comment:
RSPEC-2176: Class has same name as parent
<!-- lookout footnote separator -->
_Comment made by the analyzer sonarcheck_analyzer._